app/assets/javascripts/admin/templates/customize-themes-show.hbs

```diff
@@ -126,6 +126,15 @@
         </span>
       {{/if}}
     </div>
+  {{else}}
+    <div class="control-unit">
+      <span class="heading">{{i18n "admin.customize.theme.uploader"}}</span>
+      <span>
+        {{#user-link user=model.user}}
```

Comment:
This is the original creator, right? Someone else [could upload another version of the theme later](https://github.com/discourse/discourse/blob/501b19b6e005f00ffbaa126965b80bad3eaca47b/app/models/remote_theme.rb#L39) (normally via theme cli)

Should we change this to "Created by {{user}}"?

Reply:
Yes wasn't sure about this... Guess created is slightly better, will do the change :+1: